src/styles/blocks/header.scss

```diff
@@ -0,0 +1,58 @@
+.header {
+  padding: 0 50px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  @include flex(center, space-between);
+}
+
+.logo__link {
+  display: flex;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  &__list {
+    flex-wrap: wrap;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    @include flex(center, center);
```

Comment:
Similar to the previous comment, ensure that the `flex` mixin is defined and correctly implemented in your project.